src/client/src/Web/index.tsx

```diff
@@ -33,18 +35,20 @@ export const WebApp: React.FC = () => {
             </>
           )}
         />
-        <Route
-          exact
-          path={ROUTES_CONFIG.credit}
-          render={() => (
-            <>
-              <DisconnectionOverlay />
-              <Suspense fallback={<Loader />}>
-                <MainCreditRoute />
-              </Suspense>
-            </>
-          )}
-        />
+        {canDisplayCredit && (
```

Comment:
suggested a default path to 'catch' the user if a route is unavailable, like here